ghcup-prereleases-0.0.8.yaml

```diff
@@ -1406,14 +1406,129 @@ ghcupDownloads:
+            '>= 20': &ghc-9101alpha1-x86_64-ubuntu2004
+              dlHash: 6fc4298fe60d8910eeef03d7c5eb179677e7822378ec03ad9b7237551a578122
+              dlOutput: ghc-9.10.0.20240313-x86_64-linux-ubuntu20_04.tar.xz
+              dlSubdir: ghc-9.10.0.20240313-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240313/ghc-9.10.0.20240313-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            unknown_versioning: *ghc-9101alpha1-x86_64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc-9101alpha1-x86_64-ubuntu1804
+            unknown_versioning: *ghc-9101alpha1-x86_64-ubuntu2004
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: f2168db089f6bb07d52292c704b849a2b21922445c4364705ec01c4b58173730
+              dlOutput: ghc-9.10.0.20240313-x86_64-linux-rocky8.tar.xz
+              dlSubdir: ghc-9.10.0.20240313-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240313/ghc-9.10.0.20240313-x86_64-rocky8-linux.tar.xz
+          Windows:
+            unknown_versioning:
+              dlHash: 3dc54cb6db1e40d4f7775dc1871f10e4e502f257eb516420fbb957917064fc2f
+              dlOutput: ghc-9.10.0.20240313-x86_64-windows.tar.xz
+              dlSubdir: ghc-9.10.0.20240313-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240313/ghc-9.10.0.20240313-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: a11b82c251864a3b85bcec9444373b3d009a0c5dcc12ab29e21706c2e399d271
+              dlOutput: ghc-9.10.0.20240313-aarch64-darwin.tar.xz
+              dlSubdir: ghc-9.10.0.20240313-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240313/ghc-9.10.0.20240313-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 10e793e5227185a5e8b166fe1494953cadf975a1d4fbe199e704c709aaede117
+              dlOutput: ghc-9.10.0.20240313-aarch64-linux-deb10.tar.xz
+              dlSubdir: ghc-9.10.0.20240313-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240313/ghc-9.10.0.20240313-aarch64-deb10-linux.tar.xz
+      viChangeLog: https://downloads.haskell.org/~ghc/9.10.0.20240313/docs/users_guide/9.10.1-notes.html
+      viReleaseDay: '2024-03-13'
+      viSourceDL:
+        dlHash: 2583dfb7d5fbb0afd375b1d61c3ba5b28eee641474919e956bb0282fe1756bdf
+        dlOutput: ghc-9.10.0.20240313-src.tar.xz
+        dlSubdir: ghc-9.10.0.20240313
+        dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240313/ghc-9.10.0.20240313-src.tar.xz
+      viTags:
+      - LatestPrerelease
+      - base-4.20.0.0
+      viTestDL:
+        dlHash: 25f6ab18123e76bc54b612a306f70707357742cffbd39b78d234d952389ef0d3
+        dlOutput: ghc-9.10.0.20240313-testsuite.tar.xz
```

Comment:
Please don't use `dlOutput` unless it actually differs from `dlUri`.. otherwise its just a lot of noise in the metadata. It's supposed to be human readable.